docs/genai/development/working-with-rag.ipynb

```diff
@@ -66,9 +51,9 @@
    "source": [
     "## Install a vector DB and dependencies\n",
     "\n",
-    "As mentioned above, this demo uses [Milvus](https://milvus.io) -&mdash; an open-source vector database designed specifically for similarity search on massive datasets of high-dimensional vectors. It can be installed via [docker-compose](https://milvus.io/docs/v2.0.x/install_standalone-docker.md) or [Kubernetes](https://milvus.io/docs/v2.0.x/install_standalone-helm.md) depending on your MLRun installation method. In this demo, Milvus has been installed in our K8s cluster and the service is available at `http://milvus.default.svc.cluster.local:19530`.\n",
-    "\n",
-    "Additionally, if you are using Ollama, it can be installed on your [local machine](https://ollama.com/download) or [Kubernetes](https://github.com/otwld/ollama-helm) depending on your MLRun installation method. In this example, Ollama is already installed and the service is available at `http://ollama.default.svc.cluster.local:11434`."
+    "As mentioned above, this demo uses [Milvus](https://milvus.io) -&mdash; an open-source vector database designed specifically for similarity search on massive datasets of high-dimensional vectors.<br>\n",
+    "It can be installed via [docker-compose](https://milvus.io/docs/v2.0.x/install_standalone-docker.md) or [Kubernetes](https://milvus.io/docs/v2.0.x/install_standalone-helm.md) depending on your MLRun installation method.<br>\n",
+    "In this demo, Milvus has been installed in our K8s cluster and the service is available at `http://milvus.default.svc.cluster.local:19530`."
```

Comment:
"In this demo, Milvus is already installed in the K8s cluster and the service is available at `http://milvus.default.svc.cluster.local:19530`."